test/e2e/workload/cronjob.go

```diff
@@ -75,6 +75,13 @@ var _ = framework.KubeDescribe("CronJob", func() {
 		By("Removing cronjob")
 		err = deleteCronJob(f.ClientSet, f.Namespace.Name, cronJob.Name)
```

Comment:
deleteCronJob is not setting any DeleteOptions and the default REST strategy for CJs is [orphaning](https://github.com/kubernetes/kubernetes/blob/ee8b663d22b39b515e7e55ca20e25a21aede3ebe/pkg/registry/batch/cronjob/strategy.go#L48) so I am wondering how does this test work?